systemtest/src/main/java/io/strimzi/systemtest/utils/kubeUtils/objects/PodUtils.java

```diff
@@ -238,6 +238,10 @@ public static void waitUntilPodLabelsDeletion(String namespaceName, String podNa
         }
     }
 
+    public static void waitForPendingPod(String namespaceName, String kafkaName, String nodePoolName) {
+        waitForPendingPod(namespaceName, kafkaName + "-" + nodePoolName);
+    }
+
```

Comment:
You don't need this, as you can pass `KafkaResource.getStrimziPodSetName()` there right?

Reply:
Thank you 🙏 good catch, yes you are right this will work exactly as desired.